tests/python/test_numpy.py

```diff
@@ -182,3 +182,14 @@ def test_numpy(a: ti.ext_arr(), b: ti.ext_arr()):
 def test_index_mismatch():
     val = ti.var(ti.i32, shape=(1, 2, 3))
     val[0, 0] = 1
+
+
+@ti.all_archs
+def test_numpy_zero():
+    @ti.kernel
+    def test_numpy(arr: ti.ext_arr()):
+        pass
```

Comment:
The test does nothing. Does this mean before this PR, such usage would crash?

Reply:
Sorry about the confusion. It actually does one thing: allocate a piece of memory in CUDA to store `arr`. However cudaMalloc fails if the number of bytes is 0.